.github/workflows/build.yml

```diff
@@ -143,23 +143,23 @@ jobs:
 
             -
                 name: Run PHPStan
-                run: vendor/bin/phpstan analyse -c phpstan.neon -l max src/
+                run: tests/Application/vendor/bin/phpstan analyse -c phpstan.neon -l max src/
 
             -
                 name: Run Psalm
-                run: vendor/bin/psalm
+                run: tests/Application/vendor/bin/psalm
 
             -
                 name: Run PHPSpec
-                run: vendor/bin/phpspec run --ansi -f progress --no-interaction
+                run: tests/Application/vendor/bin/phpspec run --ansi -f progress --no-interaction
 
             -
                 name: Run PHPUnit
-                run: vendor/bin/phpunit --colors=always
+                run: tests/Application/vendor/bin/phpunit --colors=always
 
             -
                 name: Run Behat
-                run: vendor/bin/behat --colors --strict -vvv --no-interaction || vendor/bin/behat --colors --strict -vvv --no-interaction --rerun
+                run: tests/Application/vendor/bin/behat --colors --strict -vvv --no-interaction || vendor/bin/behat --colors --strict -vvv --no-interaction --rerun
```

Comment:
Second part of the command missing  change of base path -- `vendor/bin/behat --colors --strict -vvv --no-interaction --rerun`

Reply:
Thanks, for your review :).

I disabled behat [for the original plugin](https://github.com/pledgcorporate/ecard-sylius) because I didn't have times allocate to this part.

if I have an answer to my issue on https://github.com/Sylius/PluginSkeleton/issues/204 and if I have to improve it, I will fix this.